docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+   "execution_count": null,
+   "id": "16ca5b6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_stable(\"e-\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35d8e49e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "charge_number(\"proton\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e498e01e",
+   "metadata": {},
+   "source": [
+    "[alpha particle]: https://en.wikipedia.org/wiki/Alpha_particle\n",
+    "\n",
+    "The symbols for many particles can even be used directly, such as for an [alpha particle]. To create an \"α\" in a Jupyter notebook, type `\\alpha` and press tab."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "493a2d6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_mass(\"α\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f2e5b30",
+   "metadata": {},
+   "source": [
+    "[mass number]: https://en.wikipedia.org/wiki/Mass_number\n",
+    "[half_life]: ../../api/plasmapy.particles.atomic.half_life.rst\n",
+    "[Quantity]: https://docs.astropy.org/en/stable/units/quantity.html#quantity\n",
+    "[astropy.units]: https://docs.astropy.org/en/stable/units/index.html\n",
+    "\n",
+    "We can represent isotopes with the atomic symbol followed by a hyphen and the [mass number]. In this example, [half_life] returns the time in seconds as a [Quantity] from [astropy.units]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94fcf452",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "half_life(\"C-14\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58ab7782",
+   "metadata": {},
+   "source": [
+    "We typically represent an ion in a string by putting together the atomic symbol or isotope symbol, a space, the charge number, and the sign of the charge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a65e15da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "charge_number(\"Fe-56 13+\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce4886cf",
+   "metadata": {},
+   "source": [
+    "[plasmapy.particles]: ../../particles/index.rst\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "[particle-like]: https://docs.plasmapy.org/en/latest/glossary.html#term-particle-like\n",
+    "\n",
+    "Functions in [plasmapy.particles] are quite flexible in terms of string inputs representing particles. An input is [particle-like] if it can be transformed into a [Particle].  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "05e03a8b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_mass(\"iron-56 +13\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0dd9541b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_mass(\"iron-56+++++++++++++\")"
```

Comment:
Haha

This is how I'm doing it from now on.

Reply:
+1

Reply:
I mean +13